modules/t/DumpWGAExpectedTags.t

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env perl
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+use strict;
+use warnings;
+
+use Test::More;
+
+use Bio::EnsEMBL::Test::MultiTestDB;
+use Bio::EnsEMBL::Hive::DBSQL::DBConnection;
+use Bio::EnsEMBL::Hive::Utils::Test qw(standaloneJob);
+
+use File::Temp qw/tempfile/;
+
+# Check module can be seen and compiled
+use_ok('Bio::EnsEMBL::Compara::RunnableDB::GeneTrees::DumpWGAExpectedTags');
+
+# Load test DB
+my $multi_db = Bio::EnsEMBL::Test::MultiTestDB->new( "test_master" );
+my $compara_dba = $multi_db->get_DBAdaptor( "compara" );
+my $dbc = Bio::EnsEMBL::Hive::DBSQL::DBConnection->new(-dbconn => $compara_dba->dbc);
+my $compara_db = $dbc->url;
+
+# Number of tags before calling the module
+my $sql1 = "SELECT COUNT(*) AS total_rows FROM method_link_species_set_tag";
+my $db_vals1 = $dbc->db_handle->selectrow_hashref($sql1);
+my $all_count = $db_vals1->{total_rows};
```

Comment:
```suggestion
my $num_tags_query = "SELECT COUNT(*) FROM method_link_species_set_tag";
my ($num_tags_before) = $dbc->db_handle->selectrow_array($num_tags_query);
```

Reply:
Your suggestions regarding my queries and counts are so nice and elegant :) Why do we need brackets around `$num_tags_before`?

Reply:
Because `selectrow_array` is returning an array, we are telling perl to store the first element of the array into `$num_tags_before`, rather than the whole array. If you were expecting 2 fields returned, you would do something like:
```
my ($element1, $element2) = $dbc->db_handle->selectrow_array($query);
```

Reply:
Perfectly clear, as always @carlacummins :) Thank you.